src/components/LinkChooser/LinkChooser.stories.tsx

```diff
@@ -0,0 +1,78 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { SelectionIndicatorIcon } from "@components/MenuItem/MenuItem";
+import { MenuItemContentSize } from "@components/MenuItem/MenuItemContent";
+import { Meta, Story } from "@storybook/react";
+import React, { useState } from "react";
+import { LinkChooser } from "./LinkChooser";
+import { data } from "./mock/data";
+import { templates } from "./mock/templates";
+import { LinkChooserProps, SearchResult } from "./types";
+
+export default {
+    title: "Components/Link Chooser",
+    component: LinkChooser,
+    args: {
+        placeholder: "https://example.com",
+        disabled: false,
+        clearable: false,
+    },
+    argTypes: {
+        onLinkChange: { action: "onClick" },
+    },
+} as Meta<LinkChooserProps>;
+
+const LinkChooserTemplate: Story<LinkChooserProps> = (args: LinkChooserProps) => {
+    const [newTab, setNewTab] = useState<boolean>(false);
+
+    const doesContainSubstring = (source: string, target: string) =>
+        source.toLowerCase().includes(target.toLowerCase());
+
+    const getGlobalByQuery = (query: string): Promise<SearchResult[]> => {
```

Comment:
i discussed this issue with sam and i was told that the component will not live in arcade anymore, but in the clarify repo so it can be as specific as it needs to be 👍